SEC/Securities/SecuritiesIdentification.rdf

```diff
@@ -414,17 +379,17 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-lst;ListedSecurity"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-lst;RegisteredSecurity"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-id;ListedSecurityIdentifier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-id;SecurityIdentifier"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>security registry entry</rdfs:label>
-		<skos:definition>record for a listed security in a securities repository</skos:definition>
+		<skos:definition>record for a security in a securities repository</skos:definition>
```

Comment:
not sure that "repository" is the right word here - it does not store the securities in any way.

Reply:
changed to registry